Resources/Prototypes/_ADT/Mobs/Customization/reptilian/Reptilian.yml

```
@@ -51,20 +51,20 @@
   sprites:
   - sprite: _ADT/Mobs/Customization/Reptilian/custom.rsi
     state: lineeyes
-
+# Begin Floof - M3739 - #1084
 - type: marking
-  id: EarsSylveon
+  id: EarsLovelox
   bodyPart: HeadTop
   markingCategory: HeadTop
-  speciesRestriction: [Reptilian]
+  speciesRestriction: [Reptilian, Vulpkanin, Rodentia, SlimePerson, Human, Harpy, IPC, Arachne, Felinid, Kitsune] # Floof - M3739 - #1084 - The fact this was originally restricted to unathi is a crime.
```

Hand me the dunce cap, I forgot something in a PR while I was making a character in this build to prep for release.
```suggestion
  speciesRestriction: [Reptilian, Vulpkanin, Rodentia, SlimePerson, Human, Harpy, IPC, Arachne, Felinid, Oni, Kitsune] # Floof - M3739 - #1084 - The fact this was originally restricted to unathi is a crime.
```